src/users/permissions.njk

```diff
@@ -2,20 +2,20 @@
 
 <h2 class="govuk-heading-l">Set org and space roles</h2>
 
-<h4 class="govuk-heading-s">Organisation roles</h4>
+<h4 class="govuk-heading-s">Organisation level roles</h4>
 
 <details class="govuk-details" role="group">
   <summary class="govuk-details__summary" role="button" aria-controls="details-content-0" aria-expanded="false">
     <span class="govuk-details__summary-text">
-      Org level role descriptions
+      What can these roles do?
     </span>
   </summary>
 
   <div class="govuk-details__text" aria-hidden="true">
-    <h3 class="heading-small">Org billing manager</h3>
-    <p>An Org billing manager is a point of contact for billing</p>
     <h3 class="heading-small">Org manager</h3>
     <p>An Org manager can create/delete spaces and edit user roles. The Org Managers would typically be senior staff: for example, you might choose to grant the role to your technical architect and a lead developer.</p>
```

Comment:
Please change to
<p>An org manager can create and delete spaces, and edit user roles. Org managers are typically senior team members, like a technical architect or a lead developer.</p>